src/main/java/it/pagopa/interop/probing/eservice/operations/mapping/dto/SaveEserviceDto.java

```diff
@@ -18,35 +21,39 @@
 @Builder
 public class SaveEserviceDto {
```

Comment:
It is better with a marker interface DTO. As instance public class SaveEserviceDto implements DTO.

And the interface an emtpy marker interface
public interface DTO implements Serializable

Reply:
It's ok for me to make this change. Since it's a wide topic that affects all the repository i'd create a dedicated task for it to take in charge in the next sprints